public/javascripts/help.js

```diff
@@ -0,0 +1,28 @@
+
+const completedClicked = event => {
+  const element = $( event.target )
+  const topicId = element.data( 'id' )
+
+  fetch( `/help/${topicId}`, Object.assign( {}, 
```

Comment:
Since the parameters are a new object, we don't need `Object.assign` here.  I would prefer that, since the parameters are static, we refactor those out into a constant.  (I think this is done elsewhere in the front end code, it would be nice to keep the approach consistent so that when we have a front end build tool it will be easy to refactor out into a module.)